app/controllers/application_controller.rb

```diff
@@ -1,2 +1,10 @@
 class ApplicationController < ActionController::API
+  skip_before_filter :verify_authenticity_token
+  before_filter :add_allow_credentials_headers
+
+  def add_allow_credentials_headers                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          
+    response.headers['Access-Control-Allow-Origin'] = request.headers['Origin'] || '*'                                                                                                                                                                                                     
+    response.headers['Access-Control-Allow-Credentials'] = 'true'                                                                                                                                                                                                                          
```

Comment:
ah, i hadn't know that this configures cookies as well. could you point me to the resource that discusses this? Perhaps he best practice for enabling CORS on rails app is through rack middleware? http://stackoverflow.com/a/29751501